ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -112,12 +112,16 @@ export const Alleles = React.memo(({ genotype, variant, isHemiX, warning }) =>
         content={<div><b>Warning:</b> {warning}</div>}
       />
     }
-    {genotype.numAlt >= 0 ?
+    {!variant.svType && genotype.numAlt >= 0 ?
```

Comment:
I think this would work better as ternery now
```
variant.svType ?
  <Header.Content>
    {Number.isInteger(genotype.cn) && <span>CN: {genotype.cn === (isHemiX ? 1 : 2) ? genotype.cn : <b><i>{genotype.cn}</i></b>}   </span>}
    {Number.isInteger(genotype.numAlt) && <span>{genotype.numAlt > 0 ? <b><i>X</b></i>: '-'}/{isHemiX || genotype.numAlt < 2 ? '-' : <b><i>X</b></i>}</span>}
  </Header.Content> : 
  <Header.Content>
    <Allele isAlt={genotype.numAlt > (isHemiX ? 0 : 1)} variant={variant} textAlign="right" />
    /{isHemiX ? '-' : <Allele isAlt={genotype.numAlt > 0} variant={variant} textAlign="left" />}
  </Header.Content>
```